boards/arm/stm32f3_disco/stm32f3_disco.dts

```diff
@@ -66,6 +66,7 @@
 		led0 = &green_led_6;
 		led1 = &green_led_7;
 		sw0 = &user_button;
+		can-1 = &can1;
```

Comment:
Can you change it to can-primary. #22141

Reply:
Yes, if #22141 is merged before this, I will. Otherwise the samples won't work.

Reply:
The sample does not use the alias, but the node directly.

Reply:
Some samples, yes. Others (like the canopen sample) uses the aliases.

Reply:
@henrikbrixandersen #22141  is merged.

Reply:
@alexanderwachter Fixed, thanks.